samtranslator/model/connector_profiles/profiles.json

```
@@ -352,6 +352,47 @@
             }
           }
         }
+      },
+      "AWS::Location::PlaceIndex": {
+        "Type": "AWS_IAM_ROLE_MANAGED_POLICY",
+        "Properties": {
+          "SourcePolicy": true,
+          "AccessCategories": {
+            "Read": {
+              "Statement": [
+                {
+                  "Effect": "Allow",
+                  "Action": [
+                    "geo:SearchPlaceIndexForText",
+                    "geo:DescribeMap",
+                    "geo:BatchGetDevicePosition",
+                    "geo:ListDevicePositions",
+                    "geo:GetMapGlyphs",
+                    "geo:ListTagsForResource",
+                    "geo:GetDevicePositionHistory",
+                    "geo:DescribeTracker",
+                    "geo:GetMapSprites",
+                    "geo:GetMapStyleDescriptor",
+                    "geo:ListGeofences",
+                    "geo:CalculateRouteMatrix",
+                    "geo:GetDevicePosition",
+                    "geo:CalculateRoute",
+                    "geo:GetGeofence",
+                    "geo:DescribePlaceIndex",
+                    "geo:SearchPlaceIndexForSuggestions",
+                    "geo:DescribeGeofenceCollection",
+                    "geo:SearchPlaceIndexForPosition",
+                    "geo:GetPlace",
+                    "geo:ListTrackerConsumers",
+                    "geo:DescribeRouteCalculator",
+                    "geo:GetMapTile"
```

So it's easier to read at a glance:

```suggestion
                    "geo:BatchGetDevicePosition",
                    "geo:CalculateRoute",
                    "geo:CalculateRouteMatrix",
                    "geo:DescribeGeofenceCollection",
                    "geo:DescribeMap",
                    "geo:DescribePlaceIndex",
                    "geo:DescribeRouteCalculator",
                    "geo:DescribeTracker",
                    "geo:GetDevicePosition",
                    "geo:GetDevicePositionHistory",
                    "geo:GetGeofence",
                    "geo:GetMapGlyphs",
                    "geo:GetMapSprites",
                    "geo:GetMapStyleDescriptor",
                    "geo:GetMapTile",
                    "geo:GetPlace",
                    "geo:ListDevicePositions",
                    "geo:ListGeofences",
                    "geo:ListTagsForResource",
                    "geo:ListTrackerConsumers",
                    "geo:SearchPlaceIndexForPosition",
                    "geo:SearchPlaceIndexForSuggestions",
                    "geo:SearchPlaceIndexForText"
```